js/array.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
+
+var log = ["掃除", "買い物", "散歩"];
+taskList(log);
+
+const task = prompt("新しいタスクを追加");
+addTask(task);
+taskList(log);
+
+function addTask(task) {
+  if (task) {
+    log.push(task);
+    alert("追加しました。");
+  } else {
+    alert("追加できませんでした。");
+  }
+}
+
+function taskList(log) {
+  console.log('現在持っているタスク一覧');
+  log.forEach(function(task, index) {

```

Comment:
せっかくならアロー関数で記述した方がモダンでいいと思います。
```suggestion
  log.forEach((task, index) => {
```